sdk/resources/arm-resources/src/lro/azureAsyncPolling.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+// Copyright (c) Microsoft Corporation.
```

Comment:
Not a blocking but it seems there are 2 MIT license in this file.

Reply:
yes. @deyaaeldeen has commented about it in https://github.com/Azure/azure-sdk-for-js/pull/16170#discussion_r669052333